### DISCUSSION TITLE
Going from blame-view to its commits is easy, say, "O" would open any. But ...

### DISCUSSION THREAD

**Discussion by poige** [tpope/vim-fugitive#2148] [Q&A] [answered]
... where's the _commit id_ in that new commit (diff) view? It does indicate _tree id_ indeed, but not the _commit hash_ itself.

I actually stumbled upon this when tried to revert a commit found with "blame". In blame's view it was impossible with <tt>crc</tt>/<tt>crn</tt> so I opened it and realised the described above issue.

**Comment by poige** [comment]
BTW, I found out that tree id worked just the same way, but anyways it's better to have commit hash indicated there as well.

**Comment by tpope** [comment, marked as answer]
Commit hash is in the buffer name, and you can yank it with `y<C-G>`, or stick it on the command like with `<C-R><C-G>`. Next redesign of this buffer (no idea when this will happen) I will include it in the buffer itself too.
